API-Samples/C#/ArmBased/VideoIndexerClient/utils/HttpClientUtils.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace VideoIndexingARMAccounts.VideoIndexerClient.utils
+{
+    public static class HttpClientUtils
+    {
+        public static HttpClient CreateHttpClient(string token)
+        {
+            var handler = new HttpClientHandler
+            {
+                AllowAutoRedirect = false,
+                ServerCertificateCustomValidationCallback = (_, _, _, _) => true
```

Comment:
This is to allow connecting to self signed certificates. I don't think we want to put it in our sample

Reply:
done